pkg/manifest/manifest.go

```diff
@@ -384,6 +390,18 @@ func (manifest *manifest) WritePlatformConfig(out io.Writer) error {
 	return nil
 }
 
+func (manifest *manifest) WriteResourceLimitsConfig(out io.Writer) error {
```

Comment:
What is the reasoning behind writing the resource limits out to a separate config file? They'll now exist in the cgroup files themselves, the config file, and the manifest. Why all three?

Reply:
The resource_limits file is intended to replace the inflexible schema found in platform_config. This new file should support both pod and launchable cgroups and be able to versioned easily. The code here does not meet those goals – I will change it such that it does.